README.md

```diff
@@ -1,62 +1,54 @@
 
-The minimum requirements are thus:
+See the <a href="https://spacetelescope.github.io/DRESSCode/user_manual/install/" target="_blank">documentation</a> for specific installation instructions.
+
+The minimum requirements are:
 
 - Python 3.6 or later
-- HEASoft 6.25 or later, which can be downloaded from the [HEASARC website](https://heasarc.gsfc.nasa.gov/docs/software/heasoft/download.html).
+- HEASoft 6.25 or later
 - [caldb](https://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/install.html): calibration tree
 - [wcstools](http://tdc-www.harvard.edu/wcstools/): World Coordinate System utilities
 
-*Note:* the [user manual](DRESSCode_UserManual.pdf) explains the HEASoft installation step by step. Note that DRESSCode has been written and tested for HEASoft version 6.25. A later version should work as well, but possibly some minor changes are needed to the scripts.
-
 ### Docker
 
-Docker can be used to run the software without having to complete the heasoft/caldb/wcstools installation procedure yourself or to run on windows. The image can be found on [docker hub](https://hub.docker.com/repository/docker/dresscodeswift/dresscode).
+Docker can be used to run the software without having to complete the heasoft/caldb/wcstools installation procedure yourself (or to run on windows). The image can be found on [docker hub](https://hub.docker.com/repository/docker/dresscodeswift/dresscode).
 
 To download and open an interactive shell:
 
 ```sh
+docker pull dresscodeswift/dresscode
 docker run --rm -it dresscodeswift/dresscode /bin/bash
 ```
 
 From there run the pipeline, which is located in `/opt/dresscode`
 
-You will probably want to download the data onto a persistent volume. For this, see [docker documentation](https://docs.docker.com/storage/volumes/).
-
-#### Docker Development
-
-To build the image locally:
-
-```sh
-docker build --tag dresscodeswift/dresscode:latest -f Docker/dockerfile .
-```
-
-To build the base image that includes caldb SWIFT/UVOTA and wcstools:
-
-```sh
-docker build --tag dresscodeswift/heasoft-caldb-wcstools -f Docker/heasoft-caldb-wcstools.dockerfile .
-```
-
-To build the base heasoft image, follow the Heasoft Docker [instructions](https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/docker.html).
+For more, see our <a href="https://spacetelescope.github.io/DRESSCode/user_manual/install/#docker-installation" target="_blank">docker instructions</a>.
 
 ## License
 
-This project is Copyright Association of Universities for Research in Astronomy and licensed under the terms of the BSD 3-Clause “New” or “Revised” License (see the [LICENSE.txt](LICENSE.txt) file for more information).
+This project is Copyright Association of Universities for Research in Astronomy and licensed under the terms of the BSD 3-Clause “New” or “Revised” License (see the [LICENSE](LICENSE) file for more information).
 
```

Comment:
I would like to add something somewhere for the users, saying that the software is still under development. Also, if they encounter bugs/problems that they please report it to the Issues... Not sure where the best place for this is.

Reply:
all software is under development, no need to write that.

we should have users submit an issue if they find a problem. that's actually already specified in the *notes* section of the docs

Reply:
added a section called "help"